<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QtFileSystem</class>
 <widget class="QMainWindow" name="QtFileSystem">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>728</width>
    <height>505</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QtFileSystem</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item>
     <widget class="QSplitter" name="splitterH">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>TreeView</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QFrame" name="frame">
          <property name="frameShape">
           <enum>QFrame::Panel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <item>
              <widget class="QRadioButton" name="radioFileDir">
               <property name="text">
                <string>目录和文件</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="radioOnlyDir">
               <property name="text">
                <string>仅目录</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QGroupBox" name="groupFilter">
             <property name="title">
              <string/>
             </property>
             <layout class="QHBoxLayout" name="horizontalLayout_4">
              <item>
               <widget class="QCheckBox" name="chkFile">
                <property name="text">
                 <string>文件过滤</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="comboType">
                <property name="enabled">
                 <bool>false</bool>
                </property>
                <item>
                 <property name="text">
                  <string>*,h;*.cpp;*.ui</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>*.txt</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>*.svg</string>
                 </property>
                </item>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="btnApply">
                <property name="enabled">
                 <bool>false</bool>
                </property>
                <property name="text">
                 <string>应用</string>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="horizontalSpacer_3">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>30</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QTreeView" name="treeView"/>
        </item>
       </layout>
      </widget>
      <widget class="QSplitter" name="splitter">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <widget class="QGroupBox" name="groupBox_2">
        <property name="title">
         <string>ListView</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QListView" name="listView"/>
         </item>
        </layout>
       </widget>
       <widget class="QGroupBox" name="groupBox_3">
        <property name="title">
         <string>TableView</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QTableView" name="tableView"/>
         </item>
        </layout>
       </widget>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox_4">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>70</height>
       </size>
      </property>
      <property name="title">
       <string/>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QLabel" name="labFileName">
           <property name="text">
            <string>文件名：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="labFileSize">
           <property name="text">
            <string>文件大小：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="labFileType">
           <property name="text">
            <string>节点类型：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="chkFileIsDir">
           <property name="text">
            <string>节点是否目录</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <widget class="QLabel" name="labFilePath">
           <property name="text">
            <string>文件路径：</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionSetRoot"/>
   <addaction name="separator"/>
   <addaction name="actionQuit"/>
  </widget>
  <action name="actionSetRoot">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icon_tree/jiedian.svg</normaloff>:/icon_tree/jiedian.svg</iconset>
   </property>
   <property name="text">
    <string>设置根目录</string>
   </property>
   <property name="toolTip">
    <string>设置根目录</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icon_tree/exit.svg</normaloff>:/icon_tree/exit.svg</iconset>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
   <property name="toolTip">
    <string>退出</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionQuit</sender>
   <signal>triggered()</signal>
   <receiver>QtFileSystem</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>363</x>
     <y>252</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
